inception-project/inception
# #950 - Add LAPPS Grid external recommender support

**What's in the PR**
* Use DKPro Version of io-lif

**How to test manually**
- Wait for the corresponding DKPro PR to be merged
- Enable Lapps Grid recommender in the `application.properties`
- Create a project, add documents and add a Lapps grid LingPipe recommender on NER
- It should show NER recommendations

**Automatic testing**

**Documentation**
